tests/scripts/execute_docker_test_env.sh

```diff
@@ -19,4 +19,5 @@ SCRIPT_DIR="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
 docker container rm -f -v r_test || true
 IMAGE_NAME="$("$SCRIPT_DIR/construct_docker_image_complete_name.sh" "${COMBINED_R_VERSION}")"
 PROJ_DIR="$(realpath "$SCRIPT_DIR/../..")"
-docker run  --network host --name r_test -v "$PROJ_DIR":"$PROJ_DIR" -t "$IMAGE_NAME" "$PROJ_DIR" "$VARIANT"
\ No newline at end of file
+CERTIFICATE_PATH=${R_TESTS_CERTIFICATE_PATH:-/tmp/certificate/}
+docker run  --network host --name r_test -v "$PROJ_DIR":"$PROJ_DIR" -v "$CERTIFICATE_PATH":"/certificate/" --add-host exasol-test-database:127.0.0.1 -t "$IMAGE_NAME" "$PROJ_DIR" "$VARIANT"
```

Comment:
Uff, this is a handful, a bit explanation would be helpful

Reply:
Done